client/lib/wpcom-undocumented/lib/undocumented.js

```diff
@@ -280,15 +280,15 @@ Undocumented.prototype.createInviteValidation = function( siteId, usernamesOrEma
  * @param {Function} fn The callback function
  * @api public
  */
-Undocumented.prototype.settings = function( siteId, method, data, fn ) {
+Undocumented.prototype.settings = function( siteId, method = 'get', data = {}, fn ) {
```

Comment:
Oh, interesting, I hadn't thought about this for #9116. While I don't think anything existing will break as a result of the changes there, if there are undocumented endpoints expecting a callback function to be passed, they might similarly need to be updated.